Bottom: 2b5befc716275c1fb31152b1f27cea2ebbabad82
Top:    5c1e94bc28e2844ff2cb9edb8047004b0e1c95ca
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:53:08 +0000

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index b0e6f31..24b21f2 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -36,6 +36,7 @@ src/move.c
 src/netlist.c
 src/parse_y.c
 src/polygon.c
+src/pour.c
 src/rats.c
 src/report.c
 src/res_parse.c
